src/js/components/collaborationPricing.js

```diff
@@ -11,20 +11,24 @@ const CollaborationPricing = (() => {
     const compiledCategories = categories.map(category => `<li>${category}</li>`).join("\n");
     const compiledPlans = plans.map((plan) => {
       const {name, price, period, benefits} = plan;
-      const compiledPlanBenefits = benefits.map((benefit) => {
+
+      const compiledPlanBenefits = benefits.map((benefit, index) => {
+        const name = categories[index];
         if (typeof benefit === 'boolean' && benefit) {
-          return `<li class="tick">&#10003</li>`;
+          return `<li><span class="desktop-benefit tick">&#10003</span><span class="mobile-benefit available-benefit">&#10003 ${name}</span></li>`;
```

Comment:
Linijki 18, 20, 22, są nieco przy długie przez co ciężko się je czyta. Korzystając z dobrodziejstwa _templete strings_ można przedstawić te fragmenty w kilku liniach np.
```
<li>
  <span class="desktop-benefit tick">&#10003</span>
  <span class="mobile-benefit available-benefit">&#10003 ${name}</span>
</li>
```
Zastanawiałem się czy można też jakoś całą tą sekwencję ifów zastąpić jednym returnem, ale niestety widzę, że to by tylko bardziej skomplikowało, więc niech zostanie.